indico/modules/events/__init__.py

```diff
@@ -256,3 +257,31 @@ def _event_cloned(old_event, new_event, **kwargs):
 @template_hook('event-references-list')
 def _inject_event_references(event, **kwargs):
     return render_template('events/management/reference_list.html', event=event, references=event.references, **kwargs)
+
+
+@template_hook('event-location-field')
+def _render_location_field(event, **kwargs):
+    from indico.modules.events.forms import EventLocationForm
+
+    form = EventLocationForm(obj=FormDefaults(location_data={'inheriting': False}))
+    tpl = get_template_module('forms/_form.html')
+    return tpl.form_row(form.location_data, skip_label=True)
+
+
+@template_hook('event-location')
+def _render_event_location(event, **kwargs):
+    return render_template('events/management/event_location.html', event=event.as_event)
+
+
+@template_hook('event-person-links')
+def _render_event_person_links(event, **kwargs):
+    return render_template('events/management/event_person_links.html', event=event.as_event)
+
+
+@template_hook('event-person-links-field')
+def _render_event_person_link_list_field(event, **kwargs):
+    from indico.modules.events.forms import EventPersonLinkForm
+
+    form = EventPersonLinkForm(event_type=kwargs.pop('event_type'))
```

Comment:
if you use this argument make it a real argument - `**kwargs` is just there to make it future-proof when adding new arguments to the call in the template